### PR TITLE
add aliens into the interface

### PR DESCRIPTION
**Pull request by cristianfchq**

### CODE REVIEW

**Review by GustavoZHA — APPROVED** [review]

**Review by danielaSantaCruz — APPROVED** [review]

File: src/main/java/org/fundacionjala/at13/spaceinvaders/GameBoard.java [hunk 1]
```diff
@@ -15,6 +16,12 @@
     private static final int SIZE_WINDOW_WIDTH = SCALE_WIDTH * SIZE_IMAGE;
     private static final int SIZE_WINDOW_HEIGHT = SCALE_HEIGHT * SIZE_IMAGE;
     private static final int NUMBER_HUNDRED = 100;
+
```
Comment (JRojas999) [inline src/main/java/org/fundacionjala/at13/spaceinvaders/GameBoard.java:19]: this blank space we should eraser.

File: src/main/java/org/fundacionjala/at13/spaceinvaders/GameBoard.java [hunk 2]
```diff
@@ -15,6 +16,12 @@
...
+    private ArrayList<Alien> aliens;
+    private AlienGroup alienGroup;
+    public static final int NUMBER_OF_ALIEN_COLUMNS = 5;
+    public static final int NUMBER_OF_ALIEN_ROWS = 3;
+
```
Comment (JRojas999) [inline src/main/java/org/fundacionjala/at13/spaceinvaders/GameBoard.java:24]: this blank space we should eraser too.

File: src/main/java/org/fundacionjala/at13/spaceinvaders/GameBoard.java (hunk 1 above)
Comment (cristianfchq) [reply src/main/java/org/fundacionjala/at13/spaceinvaders/GameBoard.java:19]: Ok, I'll change

File: src/main/java/org/fundacionjala/at13/spaceinvaders/GameBoard.java (hunk 2 above)
Comment (cristianfchq) [reply src/main/java/org/fundacionjala/at13/spaceinvaders/GameBoard.java:24]: ok

**Review by JRojas999 — APPROVED** [review]

**Pull request merged**